nbs/examples/Getting_Started.ipynb

```diff
@@ -0,0 +1,435 @@
+{
```

Comment:
<p>I think we should release to main. This might be confusing to a new user. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>